sqlglot/parser.py

```diff
@@ -606,7 +609,8 @@ class Parser(metaclass=_Parser):
         "PARTITION BY": lambda self: self._parse_partitioned_by(),
         "PARTITIONED BY": lambda self: self._parse_partitioned_by(),
         "PARTITIONED_BY": lambda self: self._parse_partitioned_by(),
-        "PRIMARY KEY": lambda self: self._parse_primary_key(),
+        "PRIMARY KEY": lambda self: self._parse_primary_key(in_props=True),
```

Comment:
is this override safe? or is this clickhouse specific

Reply:
AFAIK `PRIMARY KEY` in table properties is CH-specific

Reply:
what about teradata?

Reply:
Dunno, do you have tests for it? Or docs? Will check it out.

Reply:
https://www.mysqltutorial.org/mysql-primary-key/

what about mysql?

Reply:
Not in props, I only changed `PRIMARY KEY` parsing in the "after schema" part.
Mysql is still "in the schema".

Reply:
Teradata doesn't have `PRIMARY KEY` in props too, only in schema.